tests/conftest.py

```diff
@@ -1,18 +1,67 @@
 """Define global fixtures."""
 
-from os.path import join
-
-import pytest
+from os.path import abspath, dirname, join
+from pickle import load as _load
 
 from cobra import Metabolite, Model, Reaction
-from cobra.test import create_test_model, data_dir
+from cobra.io import read_sbml_model
 from cobra.util import solver as sutil
 
 
 try:
-    from cPickle import load as _load
+    import pytest
+    import pytest_benchmark
 except ImportError:
-    from pickle import load as _load
+    pytest = None
+
+
+cobra_directory = abspath(join(dirname(abspath(__file__)), ".."))
+cobra_location = abspath(join(cobra_directory, "src"))
+data_dir = join(cobra_directory, "tests", "data", "")
```

Comment:
At the top, `from pathlib import Path`, then as mentioned before, move the data directory from `src/cobra/data` to `tests/data` (with the exception of the SBML files that should remain there or be in both places).

```suggestion
data_dir = Path(__file__).parent / "data"
```